airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml

```diff
@@ -0,0 +1,331 @@
+page_size: 50
+
+schema_loader:
+  type: JsonSchema
+  file_path: "./source_sendgrid/schemas/{{ options.name }}.json"
```

Comment:
what about calling it `values` instead of `options`, since semantically they are just values and not really configuration options?

Reply:
agreed. [keeping track of this issue here](https://github.com/airbytehq/airbyte/issues/15581)